### PR TITLE
@docs(@angular/cli): Added some default values to ng build

### PR DESCRIPTION
**Pull request by johnpapa**

### CODE REVIEW

File: docs/documentation/build.md [hunk 1]
```diff
@@ -65,7 +66,7 @@ All builds make use of bundling, and using the `--prod` flag in  `ng build --pro
 or `ng serve --prod` will also make use of uglifying and tree-shaking functionality.
 
 ## Options
-`--aot` Build using Ahead of Time compilation.
+`--aot` Build using Ahead of Time compilation. Defaults to false.
```
Comment (clydin) [inline docs/documentation/build.md]: This is false in Dev and true in prod

File: docs/documentation/build.md [hunk 2]
```diff
@@ -75,11 +76,11 @@ or `ng serve --prod` will also make use of uglifying and tree-shaking functional
 
 `--dev` Build target and environment to development.
 
-`--output-path` (`-op`) path where output will be placed
+`--output-path` (`-op`) path where output will be placed.
 
 `--environment` (`-e`) Defines the build environment.
 
-`--extract-css` (`-ec`) Extract css from global styles onto css files instead of js ones.
+`--extract-css` (`-ec`) Extract css from global styles into css files instead of js ones. Defaults to false.
```
Comment (clydin) [inline docs/documentation/build.md]: This is also false in Dev and true in prod

**Comment by hansl** [comment]
Your commit message is wrong. It should read:

> `docs(@angular/cli): ...`

(`docs(...)` instead of `@docs(...)`).

**Comment by filipesilva** [comment]
Closing because we've since added default information to the wiki pages.

**Pull request closed without merging**